HealthcareSystem/PatientWebApp/wwwroot/js/unpublished_feedback.js

```diff
@@ -1,4 +1,30 @@
-﻿$(document).ready(function () {
+﻿var jmbg = "";
+$(document).ready(function () {
+	var token = window.localStorage.getItem('token');
+	if (token != null) {
+		$.ajax({
+			url: "/api/user/logged",
+			type: 'GET',
+			dataType: 'json',
+			processData: false,
+			contentType: 'application/json',
+			data: JSON.stringify(token),
+			success: function (loggedUser) {
+				if (loggedUser.role != "Admin") {
+					alert('Access denied!');
+					return;
+				}
+				jmbg = loggedUser.jmbg;
```

Comment:
Also, why are you saving this? I feel like you should just make one JS file with all the JWT utility methods you'll need everywhere (things like, read the JMBG or the username from the JWT, check if there's a JWT saved and redirect if no one is logged in or the role is wrong, get the Bearer string for when you need it for AJAX, and so on), and just include that JS file and use it wherever you need it.